Things.md

```diff
@@ -0,0 +1,117 @@
+
+* Text
+* Character
+* Solid
+* Scenery 
+
+GroupHoldr contains an Array for each of the groups; each Array contains all the Things of that type currently in the game. Things are added to their respective group when added to the game and removed when they die. For example, in `FSM`, solid objects like floors and blocks would be stored in the Solid group. 
+
+### Getting Groups  
+The groups are accessible both by static name and via passing in a String.  
+
+```javascript
+// Returns the Solid group
+FSM.GroupHolder.getGroup("Solid");
+
+// Returns the first Solid
+FSM.GroupHolder.getGroup("Solid")[0];
+FSM.GroupHolder.getSolid(0);
+```
+
+More can be read on GroupHoldr on its [Readme](https://github.com/FullScreenShenanigans/GroupHoldr/blob/master/README.md).
+
+# Manipulating Things  
+
+## Shifting Things  
+A Thing's placement on the screen is based on its `top`, `right`, `bottom`, and `left` properties. These define the bounding box for a Thing. A Thing is basically just a rectangle, with its height, width, and location defined by these properties.  
+The coordinate system in which these Things are placed has an origin at the top left corner (0,0). Things are also scaled from these raw measurements to actual in-game measurements using a predefined `unitsize` (i.e. `unitsize` of 1 means no scaling, `unitsize` of 2 means in-game measurements are doubled).  
+There are a number of ways to move Things, all of which are done by manipulating these 4 properties.  
+```javascript
+// Instantiates Things used in the following code samples.
+var thing = FSM.GroupHolder.getSolid(0);
+var otherThing = FSM.GroupHolder.getSolid(1);
+```
+
+#### Shift  
+You can use the shift functions `shiftVert` and `shiftHoriz`. These move the Thing up and down or left and right.
+```javascript
+// Shifts a Thing vertically by 1.
+FSM.shiftVert(thing, 1);
+
+// Shifts a Thing horizontally by 1. 
+FSM.shiftHoriz(thing, 1);
+```
+
+#### Set  
+The `set` function (`setTop`, `setRight`, `setBottom`, or `setLeft`) moves the Thing to line up with the bounding property.
+```javascript
+// Sets the top bound of a Thing to 10. 
+FSM.setTop(thing, 10);
+
+// Sets the left bound of a Thing to 10. 
+FSM.setLeft(thing, 10);
+
+// Sets the bottom bound of a Thing to 10. 
+FSM.setBottom(thing, 10);
+
+// Sets the right bound of a Thing to 10. 
+FSM.setRight(thing, 10);
+```  
+
+#### Midpoints  
+It's possible to set the Thing's midpoint using `setMidX`, `setMidY`, and `setMid` (the latter of which sets both the `x` and `y` midpoints). These functions make it so the Thing is centered on the given `x` and `y`.  
+```javascript
+// Sets both x and y midpoints to 1 and 1, respectively. 
+FSM.setMid(thing, 1, 1);
```

Comment:
Use different numbers. It's a little easier to glance and remember that the first is x and the second is y.